newcomers-tutorial.md

```diff
@@ -122,7 +119,7 @@ approaches you used, you may want to use one of the alternative build methods:
 
 Hadrian is GHC's new build system, based on [Shake](https://shakebuild.com/).
 It is pretty stable already and should work just fine, but if you run into
-issues regardless, it may not be your fault.
+issues, it might help to read the below.
```

Comment:
That's fine. I took out "it may not be your fault" because it sounds like the default is to blame them! (in other words, it could be construed as slightly aggressive language). One of my favourite things about the original that this text is replacing is that it had a kind of fire-side chat feel to it. It was quite friendly language.

Maybe I'll change it to "if you run into issue, please don't assume it's your own fault."

Reply:
Update: okay I reworked it a bit, but I'm not *super* happy with the result yet.